homeassistant/components/blebox/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "BleBox devices",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/blebox",
-  "requirements": ["blebox_uniapi==2.0.2"],
+  "requirements": ["blebox_uniapi==2.1.0"],
```

Comment:
Do we need to bump the version to remove the air quality platform? Otherwise please move to a separate PR.

Reply:
Yes, the version needs to be bumped as in blebox_uniapi `AirQuality` becomes part of the sensors there.

Reply:
Ok.